crates/re_time_panel/src/lib.rs

```diff
@@ -394,23 +409,45 @@ impl TimePanel {
             format!("{last_path_part}/") // show we have children with a /
         };
 
-        let collapsing_header_id = ui.make_persistent_id(&tree.path);
+        let collapsing_header_id = ui.make_persistent_id(ui.id().with(&tree.path));
```

Comment:
same misunderstanding of what `make_persistent_id` does. I use "got-to-source" all the time in VSCode to remind myself what functions actually do